build.cake

```diff
@@ -545,6 +548,62 @@ Task("PackageZip")
     Zip(ZIP_IMG, zipPath);
 });
 
+Task("InstallSigningTool")
+    .Does(() =>
+    {
+        var result = StartProcess("dotnet.exe", new ProcessSettings {  Arguments = "tool install SignClient -g" });
```

Comment:
What do you think of installing to a local tool path instead of globally?
Also, how does this line behave when SignClient is already globally installed?

Reply:
I think you're probably right, I originally did it that way. I did it this way because it is almost always run on a build machine. If it is already installed, the result code is non-zero, but I don't fail on that. I'll make the change to a local install.